lab-bundle-spec.md

```diff
@@ -264,6 +277,34 @@ No additional attributes
 > **NOTE:** A draft of the `aws-account` resource type was previously specified
 > in this document. See [previous version](https://github.com/CloudVLab/qwiklabs-content-bundle-spec/blob/93896ced4ae5b543132d7a10d838ac17bd5ae3e1/lab-bundle-spec.md) for details.
 
+#### Display Outputs
+
+Specify which resources properties are given to the lab taker.
+
+Not all details of the lab environment should be exposed to the lab taken. For example, a lab may involve a GCP project and two GCP users. The lab taker is expected to log into GCP as one user, and manipulate the IAM privileges of the other. Since the lab taker is not expected to log in to the second user, there is no reason to display the GCP user's password and doing so may be distract.
+
+attribute | required | type   | notes
+----------| -------- | ------ | --------------------------------------
+label     | ✓        | string | How the property will be referenced within the script.
+reference | ✓        | string | A reference to a value, in the form `[RESOURCE].[PROPERTY]`, which will be obtained from the running resource and passed into the script.
+
+```yml
+environment:
+  display_outputs:
+    - label:
+        locales:
+          en: "GCP Project"
+      reference: primary_project.project_id
+    - label:
+        locales:
+          en: "Username"
+      reference: primary_user.username
+    - label:
+        locales:
+          en: "Password"
+      reference: primary_user.password
+```
+
```

Comment:
add something like "See each resource type's "Valid custom property references" section for the valid references."?

Reply:
I like it. :+1: 

```suggestion
See each resource type's "Valid custom property references" section for the valid references.
```